Automattic/wp-calypso
# Sites Dashboard: update the results number on each tab

#### Proposed Changes

* After searching, update the number inside the tabs badge.

We need to filter all sites with the text query and then split the sites into the different statuses.

#### Testing Instructions

* Go to /sites-dashboard
* Insert a text to search
* Observe the number is correctly updated in each tab.

#### Screencast


https://user-images.githubusercontent.com/779993/180218027-7faa48c7-ed48-437f-9c17-4d2570fc1ae2.mp4



#### Pre-merge Checklist


Solves #65606

Closes https://github.com/Automattic/wp-calypso/issues/65169.
